Assets/Scripts/Managers/BackgroundMusicManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class BackgroundMusicManager : SingletonBehaviour<BackgroundMusicManager>
+{
+	[Tooltip( "Seconds it takes to fade out a song" )]
+	[SerializeField] float fadeOutTime = 0f;
+	[SerializeField] float fadeInTime = 0f;
```

Comment:
It's nice to have the control. I currently have the fade out at 5 seconds and the fade in at 4 seconds and it sounds pretty nice. Worst case scenario you adjust two numbers instead of one. Added the second tooltip.